src/assets/icons/ic_dm_anon.js

```diff
@@ -17,8 +17,8 @@ function Ic_Dm_Anon(props) {
   return (
     <Svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
       <Path
-        fill-rule="evenodd"
-        clip-rule="evenodd"
+        fillRule="evenodd"
+        clipRule="evenodd"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are correct and follow the JSX syntax for attribute names. In JSX, camelCase property naming convention is used instead of the HTML attribute names. Therefore, `fill-rule` becomes `fillRule` and `clip-rule` becomes `clipRule`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->